cartridges/int_adyen_controllers/cartridge/scripts/payment/processor/ADYEN_CREDIT.js

```diff
@@ -115,9 +94,13 @@ function Authorize(args) {
             PlaceOrderError: (!empty(args) && 'AdyenErrorMessage' in args && !empty(args.AdyenErrorMessage) ? args.AdyenErrorMessage : '')
         };
     }
-
-    if (result.IssuerUrl != '') {
+ 
+    if (result.RedirectObject != '') {
```

Comment:
add checks if result.PaymentData, result.RedirectObject.url result.RedirectObject.data.PaReq and result.RedirectObject.data.MD  exists. If not log errors